exla/lib/exla/op.ex

```diff
@@ -212,6 +212,29 @@ defmodule EXLA.Op do
     %{op | ref: result_ref}
   end
 
+  def is_infinity(%{ref: ref} = op, {:c, _}) do
+    re_part = ref |> EXLA.NIF.real() |> unwrap!() |> EXLA.NIF.is_infinity() |> unwrap!()
+    im_part = ref |> EXLA.NIF.imag() |> unwrap!() |> EXLA.NIF.is_infinity() |> unwrap!()
+
+    result_ref = EXLA.NIF.bitwise_or(re_part, im_part, {}) |> unwrap!()
+
+    %{op | ref: result_ref}
+  end
+
+  def is_infinity(op, type) do
+    %{ref: ref} =
+      case type do
+        {t, _} when t in [:f, :bf] ->
+          op
+
+        _ ->
+          convert_element_type(op, Nx.Type.to_floating(type))
```

Comment:
Sorry, I should have caught this in the nan review. :) If the tensor is not f or bf, shouldn't we simply multiply it by zero instead?

Reply:
Yeah, that makes sense :)